### DISCUSSION TITLE
What tools can I use to integrate CLS and RLS in xtable?

### DISCUSSION THREAD

**Discussion by silwalanish** [apache/incubator-xtable#501] [Q&A]
What tools can I use to integrate CLS and RLS in xtable? 

If anyone can point to good starting point would be much appreciated.

**Comment by the-other-tim-brown** [comment]
Hi @silwalanish, what is CLS and RLS? I tried googling but the results seemed unrelated 😅

**Reply by silwalanish** [reply]
@the-other-tim-brown  Sorry for not elaborating.

CLS := Column Level Security
RLS := Row Level Security

Basically, if I want to implement column masking and row filtering through access control. What options do I have?

**Reply by the-other-tim-brown** [reply]
XTable is not a table format so you would need to use the options that the table formats are providing today. I am not familiar with these options but if you can attach some material for me to read up on, then we can see how XTable fits into the picture.
